common.mk

```diff
@@ -0,0 +1,102 @@
+#
+# Common definition to all platforms
+#
+
+BASH := $(shell which bash)
```

Comment:
Shouldn't be needed, I think I added that for "better safe than sorry". But .. when I'm thinking about it, doesn't `which` just use what is in your `PATH`? I.e, if yes, then would be really useless to have. I mean "`which`"  isn't something that is tracked with some internal database (like the `locate` command)?